posydon/utils/common_functions.py

```diff
@@ -195,52 +195,54 @@ def rzams(m, z=0.02, Zsun=0.02):
     return r
 
 
-'''
-
-
-Receives:
-q ->
-a_orb ->
-
-Returns:
-RL -> Roche lobe radius in similar units as a_orb
-'''
-
-
 def roche_lobe_radius(m1, m2, a_orb=1):
     """Approximate the Roche lobe radius from [1]_.
 
     Parameters
     ----------
-    m1 : float
+    m1 : float, array of floats
```

Comment:
Could you please use the term `ndarray` instead of `array`, because python does not have a real `array` type. Some code parts already use the correct terminology. But there are some others using the wrong one, I guess that's something to correct in a different PR.